client/src/components/App.jsx

```diff
@@ -29,6 +30,10 @@ const App = (props) => {
 
   return (
     <div>
+      <div className="row justify-content-center">
+        <div className="top_bar col-8">Top Bar Goes Here</div>
```

Comment:
Why does this say "Top Bar Goes Here"? What will actually end up going here?

Reply:
The top bar is shown in the visual example but not referenced in the specifics of the project description. I reserved space for it in the layout for future updates or changes. Presumably this would be addspace.

Reply:
I think I'd prefer if it referenced that instead of just saying "Top Bar Goes Here". Maybe something like, 
/* Future ad space*/